### ISSUE TITLE
Can't get server config

### ISSUE THREAD

**Issue by andruska** [Snawoot/nth-dump#3]
```
# ./nth-dump.linux-386
NTH-DUMP: 2022/08/31 03:00:16.955156 main.go:40: can't get server config: API response verification failed: data was not found in the response. parts found: 1
```

**Comment by ve3xone** [comment]
yes the same problem

**Comment by Snawoot** [comment]
![изображение](https://user-images.githubusercontent.com/3524671/187611097-6ff2a830-624b-4cd2-be59-df314f64269b.png)

Even their original Windows desktop app is not working with API now. Apparently, they're trying to hide evidence of this: https://snawoot.github.io/stripping_nthlink_vpn_encryption/

Will try to fix.

**Comment by Snawoot** [comment]
Yup, fixed in [v1.2.0](https://github.com/Snawoot/nth-dump/releases/tag/v1.2.0)
